### DISCUSSION TITLE
getting  error during  running  build in tailwindcss

### DISCUSSION THREAD

**Discussion by vishambardutt** [tailwindlabs/tailwindcss#8464] [Help]
Hi, guys please help me  configure tailwindcss , tailwind generating css in output buthere are only around 450 lines css in tailwindcss.
below  is warning. 

warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration

**Comment by kingx619** [comment]
In your tailwind.config.js file add all your template files to the content array like this if its a .php file else change the extension of file if its css or something else...Note all paths you provide should start from root of your directory denoted by ./ [dot slash] . Tailwind engine extracts all utility classes mentioned in your template files and then adds its to your output css file 

module.exports =
    {
        content:
            [
                './views/layouts/*.php',
                './views/pages/**/*.php',
                './views/pages/*.php',
                './views/pagesError/*.php',
            ],
        theme:
            {
                extend: {},
            },
        plugins:
            [
                require('@tailwindcss/typography'),
            ],
    }
